lib/s3.py

```diff
@@ -2,8 +2,17 @@
 
 from abc import ABCMeta, abstractmethod
 from io import BytesIO
-from boto.s3.connection import S3Connection
-from boto.s3.key import Key
+import sys
```

Comment:
(I would generally import `sys`, because it contains functions and constants; I reserve partial imports for modules containing specific objects I want to use)